### PR TITLE
update issue/pr templates with up-to-date info on v2

### PR DESCRIPTION
**Pull request by dholbach**
Clarify language and information around the state of v2.

### CODE REVIEW

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @dholbach 🍰

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -8,9 +8,21 @@ assignees: ''
...
 <!--
-Please note that Flux v1 is in maintenance mode: https://github.com/fluxcd/flux/issues/3320
+# ---- NOTICE -----
+
+Flux v1 is in maintenance mode and Flux v2 is getting closer to GA.
+If you want to learn about migrating to Flux v2, please review
+<https://github.com/fluxcd/flux2/discussions/413>.
+
+As it will take longer until we get around to issues and PRs in
+Flux v1, we strongly recommend that you starting familiaring yourself
```
Comment (kingdonb) [inline .github/ISSUE_TEMPLATE/bug_report.md:18]: s/starting/get started/

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
-For more information, and details about Flux's future,
-see: https://github.com/fluxcd/flux/issues/3320
+# ---- NOTICE -----
+
+Flux v1 is in maintenance mode and Flux v2 is getting closer to GA.
+If you want to learn about migrating to Flux v2, please review
+<https://github.com/fluxcd/flux2/discussions/413>.
+
+As it will take longer until we get around to issues and PRs in
+Flux v1, we strongly recommend that you starting familiaring yourself
```
Comment (kingdonb) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: s/starting/get started/

or "start"

**Comment by kingdonb** [comment]
Oops, looks like I was too late for this and it's already been merged

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (dholbach) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: → https://github.com/fluxcd/flux/pull/3415

**Pull request merged**